developer_guide.md

```diff
@@ -147,10 +148,11 @@ export MY_POD_NAME=my-pod
 Now we are ready to run operator locally:
 
 ```sh
+kubectl create -f examples/crd/crd.yaml
```

Comment:
I think this should be executed after launching tf_operator.

Reply:
emmm, the operator assumes that the CRD is created so I think it should be run before the creation of operator 🤔

Reply:
Never mind, it is not a requirement.